pkg/kubectl/resource/builder.go

```diff
@@ -342,28 +342,29 @@ func (b *Builder) ResourceTypeOrNameArgs(allowEmptySelector bool, args ...string
 			}
 		}
 		return b
+	} else {
```

Comment:
See https://golang.org/doc/effective_go.html#if for the general recommendation - this is a general Go style.